### ISSUE TITLE
Hidden processes on start and according menu item

### ISSUE THREAD

**Issue by Speravir** [processhacker/processhacker#1047]
With version 3.0.4399 I do on program start not get displayed all active processes, but only the ones for the active user and for `NT AUTHORITY\SYSTEM`. In menu _View_ the item “Hide processes from other users” is disabled, though (no marker). I have to enable it manually (marker is displayed) which will hide the system processes and afterwards disable it to get all processes visible, i.e. also the local and network services. This is different to version 2.39.124 where all processes are shown by default.

**Comment by Speravir** [comment]
Just for the record: No change with version 3.0.4461.

**Comment by dmex** [comment]
Try resetting the application settings?

**Comment by Speravir** [comment]
Ah, I forgot it’s a test version. Resetting worked, but then I discovered the next version has been published and after the update it was again messed up. For a stable version this should not happen, it is not acceptable to loose all custom settings on every update just to get one of them working.

I leave this open for now. But if you think this is better covered by a new issue ticket feel free to close this one.

**Comment by dmex** [comment]
I'm not seeing anyone report the same problem so it's only an issue with your machine. I'm not sure why either so maybe your settings folder might be corrupted. You could try deleting the %appdata%\ProcessHacker folder otherwise you'll have to figure out why your settings are not being saved.

**Comment by Speravir** [comment]
I do not have any `%appdata%\ProcessHacker` folder and so do not have any files there. When I started using the test versions I also copied the empty files `ProcessHacker.exe.settings.xml` and `usernotesdb.xml`, parts of the program ZIP package, which then have been filled with content – and I actually prefer this portable style. Then on an update I did not overwrite these (i.e. did not copy the empty versions over the changed files) until you mentioned a reset. 

BTW I also copied the whole folder including settings files to another place just for learning what happens with these, and noticed this would not leave to the odd behaviour being the reason for this issue ticket.

If I am for unknown reason the only one then I will have to live with this.
